deployment/helm/synopsys-alert/templates/alert.yaml

```diff
@@ -94,6 +98,22 @@ spec:
           name: java-keystore
           subPath: cacerts
         {{- end }}
+        {{- if .Values.postgres.dbCredentialSecretName }}
+        - mountPath: /tmp/secrets/ALERT_DB_USERNAME
+          name: db-credentials
+          subPath: ALERT_DB_USERNAME
+        - mountPath: /tmp/secrets/ALERT_DB_PASSWORD
+          name: db-credentials
+          subPath: ALERT_DB_PASSWORD
+        {{- end }}
+        {{- if .Values.postgres.dbAdminCredentialSecretName }}
```

Comment:
In the postgres.yaml file we don't appear to need to mount the secrets this way.  We just need the secret key references so would that be all that is needed for this alert.yaml file and not mounting the secrets?  If we don't need it then we don't need lines 109-116 and 156-167.